.github/workflows/stale.yml

```diff
@@ -0,0 +1,16 @@
+name: Mark stale issues and pull requests
+on:
+  schedule:
+    - cron: '0 0 * * *' # Run every day at midnight UTC
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 60 days with no activity. Remove the stale label or leave a comment or this will be closed in 7 days.'
+          stale-pr-message: 'This pull request is stale because it has been open 60 days with no activity.'
+          close-issue-message: 'This issue was closed because it has been marked as stale for 7 days with no activity.'
+          days-before-stale: 60
```

Comment:
We can configure this to not label stale PRs at all as well, but for the first pass i decided to just mark stale PRs in case we ever wanted to quickly search up stale PRs by the label for cleanup.